.tt_blacklist

```diff
@@ -1,6 +1,4 @@
 chemicaltoolbox/osra
-chemicaltoolbox/silicos-it/shape-it
```

Comment:
Ok, I will restore shape-it and strip-it back to the blacklist. And will push the changes again, so that it can be merged, and then will work on each new package in a new PR